HandsOff/HandsOffApp.swift

```diff
@@ -9,9 +9,21 @@ import SwiftUI
 
 @main
 struct HandsOffApp: App {
+    @AppStorage("onboardingComplete") private var onboardingComplete: Bool = false
+
     var body: some Scene {
         WindowGroup {
             ContentView()
+                .environment(\.colorScheme, .light)
+                .overlay {
+                    if !onboardingComplete {
+                        OnboardingView {
+                            onboardingComplete = true
+                        }
+                        .transition(.opacity)
+                        .environment(\.colorScheme, .light)
```

Comment:
Forcing light mode for now

Solely to make the onboarding video background seamless

since we seem to be hardcoding the blue gradient on ContentView.

Happy to make it match system theme